specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Create_WithPriorityMixPolicy.json

```diff
@@ -7,26 +7,32 @@
     "parameters": {
       "sku": {
```

Comment:
Is there a specific reason we are updating this version?  
If not, let's just update the latest version: 2024-07-01

Reply:
Good suggestion, just updated this to the latest the version

Reply:
this file needs to be updated: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-07-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Create_WithPriorityMixPolicy.json

This example is in 2022-11-01 folder 

You might need to rebase to main to see the 2024-07-01 folder, as it was merged very recently :)

Reply:
Good catch. I will also need to update the example files in all versions that have been released since the version this change was released in.

To have the updated example present in future releases, is it enough to add the example to latest stable version?

Reply:
Updated all example files since 2022-08-01 (when the feature was introduced) with the correct versions